yahoo-japan/road-network.json

```diff
@@ -0,0 +1,151 @@
+{
+  "type": "FeatureCollection",
+  "name": "road-network",
+  "data_layers": [],
+  "features": [
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [136.642, 35.096]
+      },
+      "properties": {
+        "place_name": "Japan",
+        "zoom": 8,
+        "description": "Major road networks and highway shields."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [141.5996707, 42.6487807]
+      },
+      "properties": {
+        "place_name": "Tomakomai, Japan",
+        "zoom": 18,
+        "description": "Traffic stops and oneway arrows."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [144.38677, 43.01335]
+      },
+      "properties": {
+        "place_name": "Kushiro, Japan",
+        "zoom": 15,
+        "description": "The city grid of Kushiro, Japan."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [135.76827, 35.49391]
+      },
+      "properties": {
+        "place_name": "Obama, Japan",
+        "zoom": 15,
+        "description": "Road exit signs."
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [139.758, 35.675]
+      },
+      "properties": {
+        "place_name": "Tokyo, Japan",
+        "zoom": 12,
+        "description": "Tokyo's dense roads network."
```

Comment:
It looks like this point is on a garden path in Hibiya park?

Reply:
This isn't in Tokyo, but its a good example of a dense road network 

`16/35.170858/137.005082`

<img width="868" alt="Screen Shot 2019-08-01 at 10 57 50 AM" src="https://user-images.githubusercontent.com/61150/62304047-408c6380-b44b-11e9-9bc6-d7d01522f3dd.png">

Reply:
Thanks for the good eye Tristen! A lot of copying and pasting caused some errors.

Reply:
> It looks like this point is on a garden path in Hibiya park?

@tristen I think I chose a random spot because it is because it is a z12 overview of Tokyo. I can just shift it over to a road!